src/ui/components/ConfirmButton/index.js

```diff
@@ -118,8 +117,18 @@ export class ConfirmButtonBase extends React.Component<InternalProps, State> {
   }
 }
 
-const ConfirmButton: React.ComponentType<Props> = compose(translate())(
-  ConfirmButtonBase,
-);
+export const extractId = (ownProps: Props) => {
+  return `confirmButton-${ownProps.className || ''}`;
```

Comment:
This doesn't feel ideal to me. Should I be passing in an id to the component? Or should I make this function optional? I didn't feel like that was a good idea either tho :/